src/FindResourceDuplicates.ps1

```diff
@@ -0,0 +1,68 @@
+﻿Write-Host "Running..."
```

Comment:
Maybe a comment like
```
# Xamarin ingests much of CoreFX code into Mono. Because they use .NET Framework assembly factoring they
# must merge some of our resx files. If the same resource ID is used for different strings, they cannot be merged.
# This script can be run periodically to detect any such collisions. They can then be resolved either by changing to
# use the same string, or by de-duplicating the ID's.
```